### ISSUE TITLE
A few /n chat things

### ISSUE THREAD

**Issue by yepidoodles** [Arckenver/Nations#152]
Totally minor, but I thought I'd point them out.

Firstly, using `/nc msg` and `/nc` then `<msg>` yields slightly different results. Using `/n chat` then writing a message keeps your normal chat prefix and makes your name yellow (Mine is /nick'd yellow, so I asked someone else to verify it). However when you use `/nc <msg>` there is no prefix and your name is white (this is better in my opinion). Again, totally minor, but my OCD makes me want to point it out.

Secondly, the message prefix for /nc is pretty long (especially if you're using /nc then writing in chat or using `/na spy`), however if you use #99 for the name of the Nation, that should help a lot. For `/na spy` I really believe that the prefix `[SpyChat]` is completely unnecessary as it shows you the Nation name, if you want to keep this, I suggest reducing it down to `[SC]` or `[SPY]`.

Lastly, another thing with `/na spy`, when someone talks in your Nation, the message comes up twice.

Some example screenshots:
[![Screenshot from Gyazo](https://gyazo.com/b949eb332f2f167dfbe18c0e6bf42dc6/raw)](https://gyazo.com/b949eb332f2f167dfbe18c0e6bf42dc6)

[![Screenshot from Gyazo](https://gyazo.com/6880597cbb99d764db3a9d7ba578adcf/raw)](https://gyazo.com/6880597cbb99d764db3a9d7ba578adcf)

Again, these are minor, and I know you have much to do which is more important on this, but in the interest of keeping this plugin awesome, I shall report everything :)

- Alice Sellars

P.S. My progress learning Java has been greatly hindered... I was going to try to fix some of the issues here but I really wanted to fix the lang formats first.

**Comment by yepidoodles** [comment]
Forgot to mention that `/na spy` does not stay on after re-logs... I have to turn it on every time I log in.

**Comment by TheoKah** [comment]
I wont be able to change the nickname color easily:

When toggling to nation chat, the messages are going through other plugins such as nucleus or essential and the nicknames are being set.
When using the `/nc` command with a message, this is not the case and I have to forge the whole message.
As there is multiple plugins that can set nicks, it is difficult to handle that part.

About the messages sent double when using `/na spy`. I'll see what can be done. I changed the prefix to "[SC]" for now.

The `/na spy` being lost on relog is intentional. I wanted to mimic towny's. We can discuss it.
I can easily add a perm node that makes you auto-join the channel channel on connect.

**Comment by yepidoodles** [comment]
It wasn't the nickname part I was pointing out, in fact, they are absolutely fine, it was the prefixes showing which bothered me (2-3 prefixes for a single msg).

I've never personally used Towny, in 1.7.10 and 1.6.4 I used a mod called MyTown, when I moved to 1.10.2, I went straight for Nations as Arckenver helped me with some projects in the past.
If you want to keep `/na spy`'s loss on relog, perhaps we can add a config or a permission to allow it to keep on relog?
